migrations/1763323173.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sudo pacman -S hyprmon
```

Comment:
can you change to use yay, and i believe the package is `hyprmon-bin`

This also requires a migration to make sure it is installed, since we change the omarchy-menu